profile/serializers.py

```diff
@@ -0,0 +1,103 @@
+from django.contrib.auth import get_user_model
+from django.utils.translation import ugettext_lazy as _
+from django.conf import settings
+from rest_framework import exceptions, serializers
+from allauth.utils import get_username_max_length, email_address_exists
+from allauth.account.adapter import get_adapter
+from allauth.account.utils import setup_user_email
+from allauth.account import app_settings as allauth_settings
+from dj_rest_auth.serializers import LoginSerializer as dj_rest_auth_LoginSerializer
+from .models import User
+
+UserModel = get_user_model()
```

Comment:
why is this line here? it doesn't seem to be used elsewhere

Reply:
After testing it, this line has nothing to do here.